src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+import './TodoForm.scss';
+import { User } from '../../types/user';
+import { Todo } from '../../types/todo';
+import { getNewTodoId } from '../../helpers/getNewPostId';
+import { getUser } from '../../helpers/getUser';
+
+interface Props {
+  todos: Todo[];
+  users: User[];
+  onSubmit: (newTodo: Todo) => void;
+}
+
+export const TodoForm: React.FC<Props> = ({ todos, users, onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('0');
+  const [submitting, setSubmitting] = useState(false);
```

Comment:
```suggestion
  const [isSubmitting, setIsSubmitting] = useState(false);
```
Boolean variable should have prefix `is` or `has`